src/resources/triton_python_backend_utils.py

```diff
@@ -341,6 +341,18 @@ def set_max_batch_size(self, max_batch_size):
         else:
             self._model_config["max_batch_size"] = max_batch_size
 
+    def set_dynamic_batching(self):
+        """Set dynamic_batching as the scheduler for the model if no scheduler 
+        is set. If dynamic_batching is already set, then no action is taken and
+        return success.
+        """
+        if "sequence_batching" in self._model_config.keys(
+        ) or "ensemble_scheduling" in self._model_config.keys():
```

Comment:
We should raise ValueError here. Also, the condition can be updated to include dynamic batching.

Reply:
Is there a better way in python to search through the same list multiple times? I had trouble finding anything on the python docs and general search.

Reply:
I think removing the `keys()` should probably make it more performant. Performance is not a huge concern here since we only auto-complete during model loading.

Reply:
@Tabrizian  @nv-kmcgill53 Why should we throw an error if user config has already provided dynamic_batching? 
A user may want to provide their own specific dynamic_batching settings via config.pbtxt. Things like max_queue_wait_time etc. 
We can just use whatever config the user has provided in this case.

Reply:
I agree with @tanmayv25. If the user has provided us with dynamic_batching as the scheduler and we wish to auto-complete the scheduler to dynamic_batching, then the logic is consistent and no action should be taken. The user provided config should pass through.

Reply:
I agree with the no-op if the config has dynamic batching settings and the user is trying to use dynamic batching. We can remove the error for this case.